README.md

```diff
@@ -25,11 +25,12 @@ _Quantus is currently under active development so carefully note the Quantus rel
 
 - If you want to contribute/ improve/ extend Quantus, join our [Discord](https://discord.gg/HB77krUE)!
 - New metrics added: [EfficientMPRT](https://github.com/understandable-machine-intelligence-lab/Quantus/blob/main/quantus/metrics/randomisation/efficient_mprt.py) and [SmoothMPRT](https://github.com/understandable-machine-intelligence-lab/Quantus/blob/main/quantus/metrics/randomisation/smooth_mprt.py) by [Hedström et al., (2023)](https://openreview.net/pdf?id=vVpefYmnsG)
-- Released a new version [here](https://github.com/understandable-machine-intelligence-lab/Quantus/releases) but note, support for Python 3.7 will be discontinued in future releases (https://devguide.python.org/versions/)
+- Released a new version [here](https://github.com/understandable-machine-intelligence-lab/Quantus/releases)
```

Comment:
rewrite to:  Released a new version [here](https://github.com/understandable-machine-intelligence-lab/Quantus/releases) with [Python 3.7 discontinued](https://devguide.python.org/versions/)